crates/core/src/transport/symmetric_message.rs

```diff
@@ -32,6 +32,17 @@ impl SymmetricMessage {
         Ok(PacketData::encrypted_with_cipher(bytes, outbound_sym_key))
     }
 
+    pub fn ack_ok(outbound_sym_key: &mut Aes128Gcm) -> Result<PacketData, bincode::Error> {
+        static SERIALIZED: OnceLock<Box<[u8]>> = OnceLock::new();
```

Comment:
Seems like `outbound_sym_key` shouldn't need to be borrowed as mutable to use it to encrypt, am I missing something?